README.md

```diff
@@ -1,3 +1,60 @@
-# ZIO keeper
+# Ziokeeper
 
-[![Gitter](https://badges.gitter.im/scalaz/scalaz-distributed.svg)](https://gitter.im/scalaz/scalaz-distributed?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![TravisCI][badge-travis]][link-travis]
+[![Gitter][badge-gitter]][link-gitter]
+
+Ziokeeper is a purely-functional, type-safe library for managing distributed state in a strong
```

Comment:
I would try to describe Ziokeeper as something like lightweight version of etcd(because Zookeeper is a little bit more than metadata store) and CRDT as first version of implementation. 

> Ziokeeper is a purely-functional, type-safe library for managing in-memory key-value state across nodes in cluster. First approach to achieve consistency in the cluster we plan to use CRDT. 

or something along those lines. Then I would set expectations on right level by listing functionality that we want to implement.

- cluster formation and management via gossip
- be able to subscribe on cluster events - join,leave
- access to value by key - probably we should list types that we are supporting as elements in maps right now 
- be able to subscribe on changes on particular value - modify, remove

I would also mentioned something about transport protocol(UDP/TCP) and serialization

Reply:
I've enumerated the project goals as suggested. Didn't want to provide much details about the implementation as we still have a lot of "rough" edges, but the points you made here should make the purpose of the library clear(er).

Reply:
agree